helm-charts/HPA.md

```diff
@@ -15,7 +15,7 @@
 
 ## Introduction
 
-`horizontalPodAutoscaler` option enables HPA scaling for relevant service components:
+`autoscaling` option enables HPA scaling for relevant service components:
 https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
```

Comment:
I would like to use  just `hpa` instead of `autoscaling`, and then open that as Horizontal Pod Autoscaling in the text. `autoscaling` is to generic and can mean horizontal or vertical pod, or cluster (node) autoscaling.

Reply:
Is cluster node autoscaling something that Helm charts would control (install rules for), and something that could be added to OPEA Helm charts later on?    And if yes, why that would not be also be under `autoscaling` option:
* `autoscaling.scaleNodes: true`
* `autoscaling.maxNodes: 99`
?

Reply:
`autoscaling.hpa` is good. We can extend the `autoscaling` in the future then, if needed.

Reply:
yes, that is the case. My point is that `autoscaling` is too generic. If this is about HPA autoscaling, lets have the `hpa` in the name. So, `hpa` or `autoscaling.hpa` are both fine.

Reply:
> `autoscaling.hpa` is good. We can extend the autoscaling in the future then, if needed.

As cluster scaling is not really an application install setting, there should be no confusion with that.

Are there also other relevant ways of scaling k8s _applications_ than HPA?

(`.enabled` -> `.hpa` change is OK to me, but I'm not yet seeing compelling reason to differ from NIM Helm charts.)

Reply:
Yeah, the NIM point is valid. Let's keep it similar.